### PR TITLE
Feature/add services gdev 1503

### PR DESCRIPTION
**Pull request by mephenor**
```
Added docker-compose config for remaining services
Added script for bucket initialization (this also closes GDEV-1505)
Moved test script out of pytest, has to be run manually now

Co-authored-by: Moritz Hahn <Moritz.Hahn@uni-tuebingen.de>
Co-authored-by: KerstenBreuer <kersten-breuer@outlook.com>
```

This PR only deals with adding services and a rudimentary check for the upload path.
Finalization of config for upload path and testing for correct state is part of GDEV-1506, which will also include adaptating some parts to reflect our staging/production environment better.

### CODE REVIEW

**Review by MoritzHahn1337 — APPROVED** [review]

File: .devcontainer/init-scripts/create_buckets.sh [hunk 1]
```diff
@@ -12,23 +14,7 @@
...
-    Defaults set for not running inside devcontainer.
-    """
-
-    inbox_bucket: str
-    object_id: str
-    submitter_pubkey: str
-
 
-CONFIG = Config()
+awslocal s3 mb s3://inbox
```
Comment (KerstenBreuer) [inline .devcontainer/init-scripts/create_buckets.sh:18]: When is the script executed? Is it manually run?

File: tests/unit/test_dummy.py [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Universität Tübingen, DKFZ and EMBL
```
Comment (KerstenBreuer) [inline tests/unit/test_dummy.py:1]: It would be still great if we could run, if we could run the tests scripts in an automated fashion within the CI.

**Review by KerstenBreuer — APPROVED** [review]

File: .devcontainer/init-scripts/create_buckets.sh (hunk 1 above)
Comment (mephenor) [reply .devcontainer/init-scripts/create_buckets.sh:18]: The `init-scripts` folder is mounted as volume under a specific location inside the localstack container (`/docker-entrypoint-initaws.d`). According to https://docs.localstack.cloud/references/init-hooks/ this is then run after the container becomes ready.

File: tests/unit/test_dummy.py (hunk 2 above)
Comment (mephenor) [reply tests/unit/test_dummy.py:1]: Sure, but this is something for 1506.
Wanted to remove the randomness and have state checks in place first

**Pull request merged**